helpers/ApiMarkdownTrait.php

```diff
@@ -69,6 +64,38 @@ protected function parseApiLinks($text)
         ];
     }
 
+    /**
+     * @param TypeDoc $type
+     * @param array $contexts
+     * @since 2.1.3
+     */
+    private function _findContexts($type, &$contexts = array())
+    {
+        $contexts[] = $type;
+
+        if ($type instanceof ClassDoc) {
+            if (!empty($type->traits)) {
```

Comment:
`if` + `foreach` should be extracted to a separate method to reuse it below in 82 and 91 lines

Reply:
I would rather remove this `if` - `foreach` will handle empty array just fine.

Reply:
@rob006 Done: cb74fc722cf86e1b7e4c83568943f18b9d9ec798